keycloak/keycloak-admin-ui
# User label cred edit

## Motivation
Adding change user label functionality as per: 
-> https://marvelapp.com/prototype/a9b33db/screen/83342098, 
-> https://marvelapp.com/prototype/a9b33db/screen/83342099,
-> https://marvelapp.com/prototype/a9b33db/screen/83342100,

## Verification Steps

1. Go to  users menu item, credentials tab and set up a new credential.
2. Select a pencil icon and enter a text in the text box and select cancel icon. Repeat but this time select a tick icon. Edit a few times to make sure that this is working as expected.
3. Verify that the changed label is displaying. Label should be blank by default.

Note: There is an issue I'm not able to narrow down yet, but the first time the user label is edited, the API is throwing "error: "Credential not found" even though the request path has the credentialId populated on saveUserLabel().

## Checklist:
